app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -74,10 +68,10 @@ If you're already in the UK under a [different type of visa](https://www.gov.uk/
 
 ### Staying in the UK long-term
 
-If you're interested in working as a teacher and applying for [indefinite leave to remain in the UK](https://www.gov.uk/indefinite-leave-to-remain), you should consider a [skilled worker visa](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration).   
+If you're interested in working as a teacher and applying for [indefinite leave to remain in the UK](https://www.gov.uk/indefinite-leave-to-remain), you should consider a [Skilled Worker visa](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration).   
```

Comment:
@laurahermionetennant92 It looks like you are pointing to a page that was withdrawn in May 2023 so I think this link needs updating?

Reply:
Ouch well spotted